dart_sys/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "dart_sys"
+version = "2.0.1"
+# authors = ["Patrik Buhring <patrikbuhring@gmail.com>"]
+edition = "2018"
+build = "build.rs"
+description = "Bindings to the dart native extensions api"
+# repository = "https://github.com/OptimisticPeach/dart-sys"
```

Comment:
Well I realize the original repo does *not* have a license! From what I have learnt (https://github.com/fzyzcjy/flutter_rust_bridge/issues/1), "no license" means "nobody is allowed to use it". So unless they put sth like a MIT license or explicitly allow us to fork it, we should not use it :/

Reply:
Check the Cargo.toml at the source repo
https://github.com/OptimisticPeach/dart-sys/blob/master/Cargo.toml#L12

It says that the _crate_ is MIT, but I think it would probably still be best to ask the author...

Reply:
In progress